packages/ChangesReloaded-Core.package/CRHistoryListItem.class/class/newWithSelector.andClass..st

```diff
@@ -4,7 +4,7 @@ newWithSelector: aSelectorSymbol andClass: aClassSymbol
 	| newHistoryListItem |
 	newHistoryListItem := CRHistoryListItem new
 		selector: aSelectorSymbol;
-		className: aClassSymbol name;
+		classNameOfChanges: aClassSymbol name;
```

Comment:
What about nameOfAffectedClass?

Reply:
Wouldn't call it `affectedClass` since this cane be mixed up with the actuall class object. Here, we only have a string, the `nameOfAffectedClass`. If it's about the PR in which this is solved: I don't care ^^